src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -45,26 +45,24 @@ public LiveStripeResponseGetter(StripeResponseGetterOptions options, HttpClient
 
   @Override
   @SuppressWarnings({"TypeParameterUnusedInFormals", "unchecked"})
-  public <T extends StripeObjectInterface> T request(
-      BaseAddress baseAddress,
-      ApiResource.RequestMethod method,
-      String path,
-      Map<String, Object> params,
-      Type typeToken,
-      RequestOptions options,
-      ApiMode apiMode)
+  public <T extends StripeObjectInterface> T request(ApiRequest apiRequest, Type typeToken)
       throws StripeException {
-    String fullUrl = fullUrl(baseAddress, options, path);
+    String fullUrl =
+        fullUrl(apiRequest.getBaseAddress(), apiRequest.getOptions(), apiRequest.getPath());
```

Comment:
Change fullUrl to take the apiRequest?

Reply:
Done in [Address codereview (deprecate, fullurl)](https://github.com/stripe/stripe-java/pull/1702/commits/a2262ab34877d7159e52618c98db992385815fff)